third_party/accessibility/BUILD.gn

```diff
@@ -7,6 +7,10 @@ import("//flutter/testing/testing.gni")
 
 config("accessibility_config") {
   visibility = [ ":*" ]
+  if (is_win) {
```

Comment:
I am not very familiar with gn but given we have this flag in base/build.gn and we will remove the include header in ax_node_data.cc, do we still need this?

Reply:
It is still needed but for a different reason. I'll update the comment.

../../flutter/third_party/accessibility/ax/ax_event_generator.h(129,21): error: 'iterator<std::input_iterator_tag, ui::AXEventGenerator::TargetedEvent, long long, ui::AXEventGenerator::TargetedEvent *, ui::AXEventGenerator::TargetedEvent &>' is deprecated: warning STL4015: The std::iterator class template (used as a base class to provide typedefs) is deprecated in C++17. (The <iterator> header is NOT deprecated.) The C++ Standard has never required user-defined iterators to derive from std::iterator. To fix this warning, stop deriving from std::iterator and start providing publicly accessible typedefs named iterator_category, value_type, difference_type, pointer, and reference. Note that value_type is required to be non-const, even for constant iterators. You can define _SILENCE_CXX17_ITERATOR_BASE_CLASS_DEPRECATION_WARNING or _SILENCE_ALL_CXX17_DEPRECATION_WARNINGS to acknowledge that you have received this warning. [-Werror,-Wdeprecated-declarations]
      : public std::iterator<std::input_iterator_tag, TargetedEvent> {
                    ^
c:\src\depot_tools\win_toolchain\vs_files\418b3076791776573a815eb298c8aa590307af63\win_sdk\bin\..\..\VC\Tools\MSVC\14.16.27023\include\xutility(4292,9): note: 'iterator<std::input_iterator_tag, ui::AXEventGenerator::TargetedEvent, long long, ui::AXEventGenerator::TargetedEvent *, ui::AXEventGenerator::TargetedEvent &>' has been explicitly marked deprecated here
        struct _CXX17_DEPRECATE_ITERATOR_BASE_CLASS iterator
               ^
c:\src\depot_tools\win_toolchain\vs_files\418b3076791776573a815eb298c8aa590307af63\win_sdk\bin\..\..\VC\Tools\MSVC\14.16.27023\include\yvals_core.h(597,49): note: expanded from macro '_CXX17_DEPRECATE_ITERATOR_BASE_CLASS'
 #define _CXX17_DEPRECATE_ITERATOR_BASE_CLASS   [[deprecated("warning STL4015: " \
                                                  ^
1 error generated.